test/anyof.test.js

```diff
@@ -218,7 +218,7 @@ test('symbol value in schema', (t) => {
     required: ['value']
   }
 
-  const stringify = build(schema)
+  const stringify = build(schema, { ajv: { strictSchema: false } })
```

Comment:
why this?

Reply:
because strictSchema is via this PR set to true by default...

Reply:
But is that needed for that test to pass?